python/cudf/cudf/_libxx/strings/slice.pyx

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
```

Comment:
instead of a slice file here, we should have a `replace.pyx` for the things in `replace`, a `substring.pyx` for the things in `substring`, etc.

Reply:
Lets just follow the libcudf structure entirely.